Makefile

```diff
@@ -21,8 +21,16 @@ piptools: ## install pip-compile and pip-sync.
 upgrade-piptools: piptools # upgrade pip-tools using pip-tools.
 	pip-compile requirements/pip-tools.in --rebuild --upgrade -o requirements/pip-tools.txt
 
+COMMON_CONSTRAINTS_TXT=requirements/common_constraints.txt
+.PHONY: $(COMMON_CONSTRAINTS_TXT)
+$(COMMON_CONSTRAINTS_TXT):
+	wget -O "$(@)" https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt || touch "$(@)"
+
 upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
+upgrade: $(COMMON_CONSTRAINTS_TXT)
 upgrade: upgrade-piptools piptools ## upgrade requirement pins.
+	sed 's/pyjwt\[crypto\]==1.7.1//g' requirements/common_constraints.txt > requirements/common_constraints.tmp
```

Comment:
Removing constraint from here. 
URL brings constraint `1.7.1` if i tried to override in locally it gives error.